scripts/genmod/main.go

```diff
@@ -143,8 +143,6 @@ func updateModule(dir string, gomod *GoMod, modules map[string]*GoMod) error {
 			args = append(args, "-go", *goVersionFlag)
 		}
 		cmd := exec.Command("go", args...)
-		cmd.Env = append(os.Environ(), "GO111MODULE=on")
-		cmd.Env = append(cmd.Env, "GOPROXY=http://proxy.invalid", "GOSUMDB=sum.golang.org https://sum.golang.org")
```

Comment:
I had to remove this, seemingly because of module version changes, which causes a download (before, the modules were in cache).
I'm not sure the reasoning for GOPROXY, and will merge to move forward. But I'm happy to rethink this also.